Lib/pickle.py

```diff
@@ -409,9 +409,9 @@ def __init__(self, file, protocol=None, *, fix_imports=True,
         """This takes a binary file for writing a pickle data stream.
 
         The optional *protocol* argument tells the pickler to use the
-        given protocol; supported protocols are 0, 1, 2, 3 and 4.  The
-        default protocol is 4. It was introduced in Python 3.4, it is
-        incompatible with previous versions.
+        given protocol; supported protocols are 0, 1, 2, 3, 4 and 5.
+        The default protocol is 4. It was introduced in Python 3.4, it
```

Comment:
This isn't quite grammatical. Either the comma in this line should be a full stop, or it should be a semicolon, or the last "it" on this line should be replaced with "and". I favour the last option.

Reply:
"and" here sounds good.